Spigot-Server-Patches/0594-Add-Destroy-Speed-API.patch

```diff
@@ -0,0 +1,22 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Ineusia <ineusia@yahoo.com>
+Date: Mon, 26 Oct 2020 11:48:06 -0500
+Subject: [PATCH] Add Destroy Speed API
+
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/block/CraftBlock.java b/src/main/java/org/bukkit/craftbukkit/block/CraftBlock.java
+index 3baf7b75c65c9beba40945ba904315251b5b7a64..514fad341b5a67cd2d401c91041f567c21280d44 100644
+--- a/src/main/java/org/bukkit/craftbukkit/block/CraftBlock.java
++++ b/src/main/java/org/bukkit/craftbukkit/block/CraftBlock.java
+@@ -757,5 +757,11 @@ public class CraftBlock implements Block {
+     public String getTranslationKey() {
+         return org.bukkit.Bukkit.getUnsafe().getTranslationKey(this);
+     }
++    
++    @Override
++    public float getDestroySpeed(ItemStack itemStack) {
++        net.minecraft.server.ItemStack nmsItemStack = CraftItemStack.asNMSCopy(itemStack);
```

Comment:
does this need to be a copy here? wouldn't a normal getHandle call work?

Reply:
`getDestroySpeed(new ItemStack(Material.STONE))` - woops we can't cast to `CraftItemStack` for its `#getHandle` anymore!

Reply:
can we check if its craft first and avoid clone if it is? while im sure its not hot, not a super fan of unnecessary cloning